frontend/src/properties/propertysections/ChartWidgetPropertySection.tsx

```diff
@@ -0,0 +1,136 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { BarChart, Chart } from 'charts/Charts.type';
+import * as d3 from 'd3';
+import {
+  BarChartD3ChartProps,
+  ChartWidgetPropertySectionProps,
+} from 'properties/propertysections/ChartWidgetPropertySection.types';
+import React, { useEffect, useRef } from 'react';
+import { PropertySectionLabel } from './PropertySectionLabel';
+
+/**
+ * Defines the content of a Link property section.
+ */
+
+export const ChartWidgetPropertySection = ({ widget, subscribers }: ChartWidgetPropertySectionProps) => {
+  var chartComponent: JSX.Element;
+  const { chart } = widget;
+  if (isBarChart(chart)) {
+    chartComponent = <BarChartD3Chart width={700} height={500} chart={chart} />;
+  }
+  return (
+    <div>
+      <PropertySectionLabel label={widget.label} subscribers={subscribers} />
+      {chartComponent}
+    </div>
+  );
+};
+
+const isBarChart = (chart: Chart): chart is BarChart => {
+  return chart.metadata?.kind === 'BarChart';
+};
+
+const BarChartD3Chart = ({ width, height, chart }: BarChartD3ChartProps) => {
```

Comment:
May be rename to `BarChart` would be enough.

Reply:
done